eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -182,39 +195,37 @@ void findsProbesWithVersionsInDifferentObjectionaries(@TempDir final Path temp)
             .with("hsh", first)
             .withVersionedProgram()
             .execute(new FakeMaven.Probe());
+        final ObjectName text = new OnVersioned("org.eolang.txt.text", "5f82cc1");
         MatcherAssert.assertThat(
             String.format(
-                "Tojos should have contained versioned object %s after probing, but they didn't",
-                ProbeMojoTest.STDOUT
-            ),
-            maven.externalTojos().contains(ProbeMojoTest.STDOUT),
-            Matchers.is(true)
-        );
-        MatcherAssert.assertThat(
-            String.format(
-                "Tojos should have contained versioned object %s after probing, but they didn't",
-                text
+                "Tojos should contain versioned objects '%s' after probing, but they didn't",
+                Arrays.asList(text, ProbeMojoTest.STDOUT)
             ),
-            maven.externalTojos().contains(text),
+            maven.externalTojos().contains(text, ProbeMojoTest.STDOUT),
             Matchers.is(true)
         );
         MatcherAssert.assertThat(
-            "First entry of tojos after probing should have contained two probed objects, but it didn't",
+            "Program tojo after probing should contain exactly two probed objects",
             maven.programExternalTojo().probed(),
             Matchers.equalTo("2")
         );
         MatcherAssert.assertThat(
-            "First entry of tojos after probing should have contained given hash, but it didn't",
+            "Program tojo after after probing should have given hash",
```

Comment:
@volodya-lombrozo "after after"

Reply:
@maxonfjvipon Please, try to leave as much suggestions as you can in a single PR review. I believe that It's a rather short PR to review it many times.  Actually it's wasting of CI/CD resources.